polynote-frontend/polynote/ui/component/notebook/comment.ts

```diff
@@ -387,7 +387,7 @@ class CommentButton extends MonacoRightGutterOverlay{
 
             this.clicked = true;
 
-            const newComment = new NewComment(commentsState, () => range)
+            const newComment = new NewComment(commentsState, () => range, this.hide.bind(this))
```

Comment:
To remove a NewComment previously, a user had to click outside of the selected area of the cell. On line 131, this click called the 'CommentButton' hide function on line 397.
I chose to call this same hide function when a user presses 'cancel' on a root NewComment.